apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette.component.scss

```diff
@@ -0,0 +1,9 @@
+:host {
+    background-color: white;
+    display: flex;
+    height: 100%;
+}
+
+.switch-view {
```

Comment:
```suggestion
.hidden {
```

Or something like that?

Reply:
that class is being used on both components
![image](https://user-images.githubusercontent.com/37185433/141829926-41327c72-6136-44d8-b4b0-f43e44b97bad.png)

so `switch-view` seems like a more proper name